public/src/main/java/com/revenuecat/purchases/models/GoogleStoreProduct.kt

```diff
@@ -20,6 +21,11 @@ data class GoogleStoreProduct(
     val productDetails: @RawValue ProductDetails // TODO parcelize?
 ) : StoreProduct, Parcelable {
 
+    override val id: String
+        get() = basePlanId?.let {
+            "$productId:$basePlanId"
+        } ?: productId
```

Comment:
`GoogleStoreProduct` now takes a `productId` and `basePlanId` and generates the unique `id` with the both of them if `basePlanId` exists